README.md

```diff
@@ -163,6 +163,18 @@ you can also import the Chartist settings scss:
 
 For more on custom styles see the [Chartist docs](http://gionkunz.github.io/chartist-js/getting-started.html#the-sass-way)
 
+## Plugins
+
+Install the plugin with Bower.
+
+In ember-cli-build.js, use the following example to add the plugin name: 
+
+```
+		'ember-cli-chartist': {
+			plugins: ['chartist-plugin-legend']
```

Comment:
Something we all basically know that's going to happen is that Bower go away (or at least be much less used). I wonder if we should future-proof this API a bit? Something more like:

```javascript
plugins: [
  {
    name: 'chartist-plugin-legend',
    type: 'bower', /* we could even assume this by default? */
     /* possible other options, e.g. override the name used to search the bower dir */
  }
],
```